Added minutes and July Call (#55)
mdn/community-meetings: 2024-06-24/minutes.md

[Ruth] Let us do the audit 1st and then we'll create a discussion which can link to the spreadsheet with the things that we want people to help us with. Yeah, cause you're right, yes we do, but I think we probably need a list first, don't we? And that we contributors can come in and take things off our list.

[Pranshu] Awesome! Looking forward to some progress in the next call.

[Ruth] Yeah.

### Is there an objective around closing issues? (@rumyra)

[Ruth] I had a question around closing issues because we have contributors closing lots of issues. I found out the answer earlier, which is there is no objective. People just wanted to come in and close loads of issues. So I guess the things. Mention here is you're gonna see a lot of activity over the next few weeks because some of our contributors are trying to reduce that issue count, which is fine. This is totally fine. Just because they want to. There was no other reason. They just wanted to do it. Which is cool.

### Inline icons [discussion](https://github.com/orgs/mdn/discussions/654#discussioncomment-9746229) (@rumyra)

> [Ruth] The inline icons discussion on code. This is probably something most relevant to you, Onkar. This is the idea of putting experimental and deprecated icons throughout the site so not just in the side bars but making sure they're clear on pages and in our reference pages and things like that.

Reviewer: ```suggestion

[Ruth] The inline icons discussion on code. This is probably something most relevant to you, Onkar. This is the idea of putting experimental and deprecated icons throughout the site, not just in the sidebars, but making sure they're clear in our reference pages and things like that.
```

> I want to see the vote on the discussion to see how maintainers feel we this should be done. But I don't think at this time it's a good idea for us to implement any way to do it because of the platform changes that are coming. So we have go through and have like, including macros and doing all of that kind of stuff and in like 3 months that'll all be mute and null because we'll be doing it in a completely different way.

Reviewer: ```suggestion
I want to see the vote on the discussion to see how maintainers feel this should be done. But I don't think at this time it's a good idea for us to implement it because of the platform changes that are coming. So we will be updating macros, but in about three months, that'll all be moot because we'll be doing it in a completely different way.
```

> [Florian] Yeah, I can also add on that's 1 of the things I realized that required inconsistent. So when I read it the side bars. Actually, I'm adding more of the where we previously did not. At them and I need to figure out if you wanna don't wanna have them.
> 
> [Ruth] Yeah, exactly.
> 
> [Florian] And, and at the same time it's it's quite easy like there's a global post processing of all links where we have this context so just injecting them is like really nothing to do.

Reviewer: ```suggestion
[Florian] Yeah, I can also add that when I redid the sidebars, I'm adding more icons where we previously did not have them. I need to figure out if we want to keep them.

At the same time, it's quite easy because there's a global post-processing of all links where we have this context so just injecting them is straightforward.
```

> [Ruth] That's what I'm trying to say on. Like I, I want people to vote. But no implementation to happen at this time, that's what I'm saying.
> 
> [Florian] I was wondering about one case I ran into and that I was not aware of like there's guide pages that have like a feature status is that
> 
> [Ruth] Yes, because they've got a BCD key. And we're using the BCD key for the baseline status.
> 
> [Florian] But the feature is it called feature status, don't get me wrong.
> 
> [Ruth] Okay. So yeah, I'm interested to see how maintainer feel about it. I think we also need to ask our users, but we can roll that into some decisions that we're gonna be making about the platform anyway. That's it for me.

Reviewer: ```suggestion
[Ruth] Yeah, that's what I'm trying to say. I want people to vote. But no implementation will happen at this time.

[Florian] I was wondering about one case I ran into and that I was not aware of — some guide pages have a feature status

[Ruth] Yes, because they've got a BCD key. We're using the BCD key for the baseline status.
```

> #### Other Topics
> 
> [Onkar] Onkar needs a review on pull request. I think we can take that offline.
> 
> [Christine] How does someone become a maintainer for MD and Web Talks?
> 
> [Pranshu] Good question.
> 
> [Ruth] It's usually, someone who has around for a long time which is helping with poor request reviews and issue triaging and those kind of activities and then we take it on a case-by-case basis.

Reviewer: ```suggestion
#### Other topics

[Onkar] I need a review on a pull request. I think we can take that offline.

[Christine] How does someone become a maintainer on MDN?

[Ruth] It's usually someone who has been around for a long time and has been helping with pull request reviews, issue triaging, and those kind of activities. We consider it on a case-by-case basis.
```

> [Pranshu] We don't have a fixed guidelines yet. But I will be building those. So I will have a clear response for you in a few weeks, but, right now, just get involved, just get, get your hands dirty with issues, PRs, just help us out. Just, just like Onkar does here. Chris has here and a lot of good folks who are on the yeah, and After some time, I'll just add you to becoming a maintainer, honestly. It's as simple as that right now. But, I will get back to you with concrete guidelines. I'll reach out to you. And, yeah, we'll take it offline. No worries.

Reviewer: ```suggestion
[Pranshu] We don't have fixed guidelines yet. I will be building those, so I will have a clear response for you in a few weeks. For now, just get involved, get your hands dirty with issues, PRs, and help us out, just like Onkar, Chris, and other folks here. After some time, I'll add you as a maintainer, it's as simple as that. I'll reach out to you, and we'll take it offline. No worries.
```

> [Onkar] I'm being asked to convert an image to a SVG here, but I'm not well-versed with SVG, how to proceed? Can we have a tool hosted somewhere to convert P and G to SVG for other contributors?
> 
> [Brian] I can help you with that one Onkar. If you want, I've been doing a lot of the HTTP diagrams in the last couple of weeks. Mostly this has been redoing them from scratch, which has been okay if they're mermaids, then we have the Source for this afterwards. And otherwise, they're just boxes and lines. And this is pretty easy to recreate so I can give you a hand with that if you want.
> 
> [Ruth] Thanks, Brian.

Reviewer: ```suggestion
[Onkar] I'm being asked to convert an image to an SVG, but I'm not well-versed with SVG. How should I proceed? Can we have a tool hosted somewhere to convert PNG to SVG for other contributors?

[Brian] I can help you with that, Onkar. I've been doing a lot of the HTTP diagrams in the last couple of weeks, mostly redoing them from scratch, which has been okay if they're mermaid diagrams, because then we have the source for them afterwards. Otherwise, if they're just boxes and lines, they're pretty easy to recreate, so I can give you a hand with that if you want.
```

> [Pranshu] Awesome. So we don't have anything else on the agenda. One last important update that I wanted to share was that our next community call will be on the 23rd of July. 2024, which you, if you haven't guessed it yet, it's, MDN will turn 19. I hope to see all of you there.
> 
> Okay. Cool, awesome. Alright, so we still have 15 min. Which I happily give you back because it's Chris' birthday. So, alright, thank you so much to everyone for joining. And see you in the next one.

Reviewer: ```suggestion
[Pranshu] Awesome. We don't have anything else on the agenda. One last important update that I wanted to share was that our next community call will be on the 23rd of July, 2024. If you haven't guessed it yet, MDN will turn 19. I hope to see all of you there.
```

# Minutes from MDN community meeting - 24th June 2024

> **Attendees:** Ruth John (@Rumyra), Chris Mills (@chrisdavidmills), Vadim Makeev (@pepelsbey), Pranshu Khanna (@pransh15), Dipika Bhattacharya (@dipikabh), Christine Belzie (@CBID2), Florian Dieminger (@fiji-flo), Claas Augner (@caugner), Florian Scholz (@elchi3)

Author: Done. ✅

> **Attendees:** Ruth John (@Rumyra), Chris Mills (@chrisdavidmills), Vadim Makeev (@pepelsbey), Pranshu Khanna (@pransh15), Dipika Bhattacharya (@dipikabh), Christine Belzie (@CBID2), Florian Dieminger (@fiji-flo), Claas Augner (@caugner), Florian Scholz (@elchi3)

Reviewer: Thank you!

**Host:** Pranshu Khanna (@pransh15)

## Intro and overview

> [Pranshu] So, hello everyone. Welcome to the June community call. I've shared the agenda in the chat. While you look at the agenda and if you're not a part of our discord community, please join the MDN community. There we have a lot of, MDN related chats. So some might interest you, some might not. And while you're on MDN spaces, make sure that you follow the Mozilla community participation guidelines.

Author: Makes sense. 👍

[Pranshu] Sounds good.

### GitHub Week - 8th July (@rumyra)
[Ruth] Next to the, the week of July the 8, th the content and community team at Mozilla will be having a GitHub week.  We do try to do these once every 6 months. I think we missed the last one and it's basically sort of going and doing a bit of spring cleaning, making sure we're flushing out still things, reviewing some of our processes and looking at the pain points again and considering those. We're not going to have everything done in one week, but it's a space just for us to be able to focus on that specifically, like the GitHub organization and the management and things like that, which we don't tend to do on our day-to-day. We're more focused on other things. So we will make an announcement in discussions. I'm assuming this week, with a few goals and what to expect. But we just wanted to let you all know just in case you see us all around for one week and we're doing lots of things. We will keep everybody informed. With the things that we are actually doing.

[Chris] I was just going to ask is that just related to kind of closing style issues or is it a wider variety of things?

> [Ruth] We will do some triaging and we will look at stale things. The thing is that that has been more of a focus on previous GitHub weeks and these days we actually have quite a lot less than we probably think we do. I think with Pranshu starting and joining last month, it's also really good opportunity just to look at the processes and the way that we do trash things, what our projects look like. Teams and access rights, all those processes kind of need just to sort of be checked and reviewed and make sure that they work for everybody.

Reviewer: ```suggestion
[Ruth] We will do some triaging and we will look at stale things. I think with Pranshu starting last month, it's a really good opportunity to look at the processes, the way that we track things, what our projects look like, and teams and access rights. All those aspects need to be sorted and reviewed periodically to make sure they're working for everybody.
```

[Onkar] Is it like inviting students to work on small tasks?

> [Ruth] Not this one. I think that is a different kind of week, which I do hope we will be able to run at some point, but I think this one will just be the core like. The people in the room who are writers here, Pranshu and we can Just start a plan for the next 6 months and then we'll do it all over again at Christmas and then all over again this time next year and just keep iterating and improving.

Reviewer: ```suggestion
[Ruth] Not this one. This one will include the core team, the writers and Pranshu. We repeat this exercise every six months for iterating over and improving our processes and guidelines.
```

[Chris] That's great. It sounds super positive because I've noticed that OWD folks have been doing quite a bit of issue triage and stuff recently. So that's great and then Some of our volunteers have as well like Josh Chen has done absolutely loads of stuff recently closing a load of stale issues. I'm also wondering whether it would be worth spending a bit of your time kind of figuring out a way to highlight issues which would be good kind of You know. Like good 1st issue type things but not necessarily that but you know just sort of this is something that can be worked on without a massive mind field hassle, go crazy and run away screaming.

[Ruth] Yeah.

[Chris] I say this not purely just for kind of you know, regular. Flow of. Volunteer contributions but also it's worth having a bunch of those in hand for when you start to get efforts like Hacktoberfest and stuff piling onto your site.

[Ruth] Yeah.

> [Chris] Because if they can't find anything obvious to work on, they tend to start making all sorts of ludicrous unnecessarily small changes just so they can get their commit numbers up, which drives everyone crazy. So it's good to preempt things like that as well. I speak from experience having suffered through this for many years when I was working on MDN at Mozilla.

Reviewer: This can perhaps be removed altogether

Author: Done. ✅

> [Ruth] Yeah, so it's our previous community manager. We haven't had one for yet, but he did set up a project specifically for contributors to come in. So I think it would be good idea to revisit that and see whether we can improve it. And Yeah, look at look at the processes for. When people come in, there was a comment recently that our contribution docs take you to get her and then take you back to the contribution docks and that flow is a little bit disjointed. So things like that I think we can look at. Say yes.

Reviewer: This can probably be combined with one of Ruth's previous comments
```suggestion
[Ruth] Our previous community manager set up a project specifically for contributors. It would be a good idea to revisit that and see whether we can improve it. We also want to improve the workflow between contribution docs on MDN and what we have on GitHub because the flow is a little bit disjointed at the moment.
```
